data/aliases.js

```diff
@@ -428,6 +433,25 @@ exports.BattleAliases = {
 	"ohmagod": "Plasmanta",
 	"zong": "Bronzong",
 
+	// ultra beasts
+	"absorption": "Buzzwole",
+	"beauty": "Pheromosa",
+	"blade": "Kartana",
+	"blaster": "Celesteela",
+	"expansion": "Buzzwole",
+	"glutton": "Guzzlord",
+	"lightning": "Xurkitree",
+	"symbiont": "Nihilego",
+	"ub01": "Nihilego",
+	"ub02absorption": "Buzzwole",
```

Comment:
Remember that there are two UB-02 and two UB-04. I guess I can remove UB-03 Lightning